clone-item/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "clone-item",
+  "version": "1.0.0",
+  "description": "",
```

Comment:
Maybe add a description (just because it's a public repo?

Reply:
Added description now.